client/src/AppBundle/Resources/translations/admin-clients.en.yml

```diff
@@ -23,6 +23,12 @@ clientDetails:
     address: Named deputy address
     contact: Named deputy contact details
 
+clientDischarge:
+  htmlTitle: Administration - Clients
+  pageTitle: Discharge deputy from client
+  warning: %clientFirstName% %clientLastName% will be permanently detached from their named deputy.
```

Comment:
Do we need to warn that associated unsubmitted data (e.g. reports, documents) will be deleted too?

Reply:
Done 👍